spec/controllers/http_status_agent_spec.rb

```diff
@@ -36,6 +36,9 @@ def f.get url
 
                        def f.set url, response, time = nil
                          sleep(time/1000) if time
+                         def response.to_hash
```

Comment:
@cantino @strugee 

I don't think going with the new ```MockResponse``` is the best way to change this mock.  The structu can be changed itself to:

```ruby
Struct.new(:status, :headers, :to_hash)
      .new(200, {}, { url: url })
```

Then you can test the url coming out.  That's all ```to_hash``` is, a method returning a hash.

Given what I've learned about how tests are normally written on huginn agents, I can throw my hat into possible devs to refactor.  I think the better approach at this point would be to use webmock to fake the web response instead of faking the result with a fake object.  Then the tests would have everything usually available when making a web call.  It's more complicated, but it leads to simpler tests in this situation.